Makefile

```diff
@@ -62,16 +70,12 @@ help:
 	@echo "- ci                 Create the python virtual environment and install requirements based on the Pipfile.lock"
 	@echo -e " \033[1mFORMATING, LINTING AND TESTING TOOLS TARGETS\033[0m "
 	@echo "- format             Format the python source code"
-	@echo "- ci-check-format    Format the python source code and check if any files has changed. This is meant to be used by the CI."
```

Comment:
Why removing this target from help ?

Reply:
The Makefile is more or less a copy of service-qrcode I think. But I can rather use another one as a template. Which one is most current? Service-wmts?

Reply:
We cannot keep in sync all makefile from all services, trying to copy paste from makefile to another without understanding it is just a bad idea. Here you have a very useful make target `ci-check-format`, simply leave it.

Reply:
The target itself is still there, it's just removed from the help. I can re-add it, no problem. It is meant to be used by CI and not really manually, but anyways we can have it documented here.

Reply:
> 

It was a lazy way of adding the LOGS_DIR, clean_logs, et. al. stuff to the Makefile in a bulk operation.
I'll delete the unit test reports related things.
We can discuss if we should have `$(REQUIREMENTS)` in the test target, or not.
Some services don't have it, some do. Guess the world will keep on turning either way :wink:

Reply:
@hansmannj  copy pasting from repo to another without understanding what's under the hood is never a good idea and in the long terms brings more problem that it solved in general. Some makefile in repo are unfortunately handling dependencies differently. In makefile there are hundreds of correct ways to handle dependencies. After looking closer we can see that here the `$(REQUIREMENTS)` dep is not needed and probably another relique of copy/pasting from another repo. So if already doing some clean up here I would suggest to do it correctly by removing totally `$(REQUIREMENTS)` as well as the `$(TIMESTAMPS)` which is not used.

Reply:
Fully agree Brice, and no worries, I did not blindly copy and paste.
But you are right, I missed the point of the unit test reports in CI, which were added but shouldn't have been.
Next time I'll pick only those parts I surely want to add and won't do a bulk copy&paste.
So, one other round of clean-up, and hopefully this PR is finished.